### PR TITLE
Controller advice

### PR DESCRIPTION
**Pull request by Shivankshikh**
Hii..
I have updated the code by enabling Controller advice to handle the exception globally.

### CODE REVIEW

**Comment by Shivankshikh** [comment]
Hii..
I have updated the code by enabling Controller advice to handle the exception globally.

File: src/main/java/com/shivankshi/emscrud/principal/UserPrincipal.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+	@Override
+	public String getPassword() {
+		// TODO Auto-generated method stub
+		return user.getPassword();
+	}
+
+	@Override
+	public String getUsername() {
+		// TODO Auto-generated method stub
```
Comment (abhijay-kumar) [inline src/main/java/com/shivankshi/emscrud/principal/UserPrincipal.java]: Remove auto generated comments

File: src/main/java/com/shivankshi/emscrud/rest/EmployeeRestExceptionHandler.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		error.setStatus(HttpStatus.NOT_FOUND.value());
+		error.setMessage(exc.getMessage());
+		error.setTimeStamp(System.currentTimeMillis());
+
+		return new ResponseEntity<>(error, HttpStatus.NOT_FOUND);
+
+	}
+
+	@ExceptionHandler
+	public ResponseEntity<EmployeeErrorResponse> handleException(Exception exc) {
```
Comment (abhijay-kumar) [inline src/main/java/com/shivankshi/emscrud/rest/EmployeeRestExceptionHandler.java:23]: ControllerAdvice is defaulting to base exception.

File: src/main/java/com/shivankshi/emscrud/principal/UserPrincipal.java (hunk 1 above)
Comment (Shivankshikh) [reply src/main/java/com/shivankshi/emscrud/principal/UserPrincipal.java]: Removed unnecessary comments.
https://github.com/Shivankshikh/EMS/pull/4/commits/5742c776530c5e4afc8838a4744781b32697a3a7

File: src/main/java/com/shivankshi/emscrud/rest/EmployeeRestExceptionHandler.java (hunk 2 above)
Comment (Shivankshikh) [reply src/main/java/com/shivankshi/emscrud/rest/EmployeeRestExceptionHandler.java:23]: Code is working fine.
Error was coming due to executing the old branch which was not updated.

**Pull request closed without merging**